packages/client/src/comp/states/Base.tsx

```diff
@@ -32,7 +32,12 @@ const NameWrapper = styled.div`
   .mui-textfield { flex: 1; }
   ${cssQuery.medium} {
     flex-direction: row;
-    .mui-textfield:nth-child(1) { margin-right: 15px; }
+    flex-flow: row wrap;
```

Comment:
Only one nit: I think the correct way to implement a "Bootstrap"-like grid is to have
- `Row` have `margin: 0 -8px`
- `Col` have `padding: 0 8px; width: 50%;` etc ...

That way, no matter the width, it will work and you don't have to use `:nth-child`.  I tried to get this to work and couldn't -- maybe for a subsequent PR?